src/credit/web.rs

```diff
@@ -64,8 +68,15 @@ pub async fn lookup_quote(
 ) -> Result<Json<LookUpQuoteReply>> {
     log::debug!("Received mint quote lookup request for id: {}", id);
 
-    let service = ctrl.quote_service.clone();
-    let quote = service.lookup(id)?;
-    let response = LookUpQuoteReply::from(quote);
-    Ok(Json(response))
+    let quote = ctrl.lookup(id)?;
+    Ok(Json(quote.into()))
+}
+
+#[allow(dead_code)]
```

Comment:
I think we can remove this allow(dead_code), since it's used in `main` (same for the admin routes)